runelite-client/src/main/java/net/runelite/client/plugins/customcursor/CustomCursorConfig.java

```diff
@@ -40,4 +42,14 @@ default CustomCursor selectedCursor()
 	{
 		return CustomCursor.RS3_GOLD;
 	}
+
+	@ConfigItem(
+		keyName = "customImage",
+		name = "Custom Image",
+		description = "Use your own image file for a cursor"
+	)
+	default File customImageFile()
+	{
+		return new File("\\");
```

Comment:
is there a better default I can use here? I didn't want to do `null`

Reply:
`null` seems fine to me, I believe that's what we use for `Color` inputs, for instance.